### DISCUSSION TITLE
AnimeMatrix gif speed

### DISCUSSION THREAD

**Discussion by maruszczyk** [seerge/g-helper#1400] [Ideas]
Hi, would it be possible to add a gif speed. Something similar to the, where we can change size now. As having both of them would be really useful for finetuning those newly downloaded gifs.

**Comment by seerge** [comment]
@maruszczyk hello, currently app reads gif speed (delay between frames in milliseconds) from gif itself and uses it.

So it should be automated.

P.S. With only limitation: minimal delay is 50ms (i.e. 20fps max), as anime matrix itself just can't refresh more often :)
